apps/ui/components/reservation/EditStep0.tsx

```diff
@@ -44,6 +38,7 @@ import { BlackButton, MediumButton } from "@/styles/util";
 import Legend from "../calendar/Legend";
 import ReservationCalendarControls from "../calendar/ReservationCalendarControls";
 import { CalendarWrapper } from "../reservation-unit/ReservationUnitStyles";
+import { eventStyleGetter } from "@/pages/reservation-unit/[id]";
```

Comment:
I'd rather not import from @pages. Could we move it to new or something `components/reservation` or `components/reservationCommon`. Something temporary is fine.

Reply:
Made a new `calendarUtils.ts` file under `components/common`.